cpp/data/installer/apparmor/apache2

```diff
@@ -0,0 +1,30 @@
+# vim:syntax=apparmor
+#include <tunables/global>
+
+/usr/sbin/apache2 flags=(attach_disconnected) {
+  #include <abstractions/base>
```

Comment:
From https://gitlab.com/apparmor/apparmor/-/wikis/QuickProfileLanguage "The # is optional and shouldn't be used in newer
profiles. "

Reply:
Has been removed now, however i could not find any example in my /etc/apparmor.d that uses `include` without `#`.